src/index.html

```diff
@@ -1418,7 +1436,16 @@ <h6>Reporting</h6>
     }
 
     function isValidAccessionID (accessionID) {
-        return /^(rs|ss)[0-9]+$/.test(accessionID);
+        return /^(rs|ss)[0-9]+$/.test(accessionID) || /^(RS|SS)[0-9]+$/.test(accessionID);

```

Comment:
If the input is `Rs` this won't work. Can you instead ignore the case or convert it to lower case before validating?